ops/ansible/roles/bfd-db-migrator/templates/bfd-db-migrator-service.sh.j2

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+export DATABASE_URL='{{ db_migrator_db_url }}'
+export DATABASE_USERNAME='{{ db_migrator_db_username }}'
+export DATABASE_PASSWORD='{{ db_migrator_db_password }}'
+
+LOGS_DIR='{{ db_migrator_dir }}/'
+
+# Either don't set this variable, or set it to one of: BENEFICIARY, CARRIER, DME, HHA, HOSPICE, INPATIENT, OUTPATIENT, PDE, SNF
+# export DATA_SET_TYPE_ALLOWED="BENEFICIARY"
+
+exec "{{ db_migrator_dir }}/bfd-db-migrator-1.0.0-SNAPSHOT/bfd-db-migrator.sh" \
+	"-DbfdDbMigrator.logs.dir=${LOGS_DIR}" \
```

Comment:
In the current db migrator app I see that we are logging to STDOUT and capturing that here with the shell redirection. Is there another log that is anticipated to be created by the application?

Reply:
Based on my conversations with Logan so far, I _don't think_ this behavior will be changing. Though, your comment is reminding me of some related things I'd like to handle in the PR. I'll reach out to @lsmitchell–maybe he has some additional thoughts from his most recent work on the product engineering side.

Reply:
Currently I'm just copying the functionality from the pipeline app, so I think it should be the same and there won't be any additional logging yet; we could add some if it is useful but no plans now. My only thing I need to check is if absent from the pipeline app, if the log output remains the same (not sure if pipeline is doing something special with capturing/redirecting the logging)